core/kazoo_number_manager/src/knm_phone_number.erl

```diff
@@ -874,6 +874,19 @@ list_attachments(PhoneNumber, AuthBy) ->
 
 %%--------------------------------------------------------------------
 %% @private
+%% @doc Sanitize phone number docs fields and remove deprecated fields
+%% @end
+%%--------------------------------------------------------------------
+-spec sanitize_public_fields(kz_json:object()) -> kz_json:object().
+sanitize_public_fields(JObj) ->
+    kz_json:delete_keys([<<"id">>
+                        ,<<"used_by">>
```

Comment:
Wasn't `<<"used_by">>` replaced by `?PVT_USED_BY` ? I thought that was replaced, so removing it to prevent future confusion.